develop_docker/development-tasks.md

```diff
@@ -0,0 +1,43 @@
+The changes:
```

Comment:
Should we delete this file?

Reply:
I'm planning on deleting this file, I don't think it is needed anymore

Reply:
still up for deletion?

Reply:
removed